components/expenses/ExpenseForm.js

```diff
@@ -302,6 +307,12 @@ const ExpenseFormBody = ({
     }
   }, [values.payeeLocation]);
 
+  React.useEffect(() => {
+    if (isMultiCurrency && dirty) {
+      formik.setFieldValue('currency', undefined);
+    }
+  }, [values.payoutMethod]);
+
```

Comment:
Let's not reset the currency if it stays the same. We don't need to reset the currency if my expense is in USD and I switch from a USD payout method to a USD payout method.

The only case where we need to reset is:
- Expense is multi-currency (see the comment above)
- **AND** the new payout method currency is different from the expense currency

If the payout method doesn't have a currency, we don't need to reset. As mentioned in the issue this is problem only affects Wise.

Reply:
I've modified the `if` statement to account for this.